### ISSUE TITLE
It's Just A Tree Demo ?

### ISSUE THREAD

**Issue by KangSpace** [ProteanHub/NCTreeView#1]
Hello John ,
     I found and deployed your treeview app for nextcloud on my nextcloud server. And I got 
![image](https://user-images.githubusercontent.com/26271023/67025762-29e7d580-f139-11e9-828d-7e20980fa7e7.png) this view. Is it only a demo tree?
     Would you continue to develop the directory tree , like this，sync the file directory as a tree in the left navigation?
![image](https://user-images.githubusercontent.com/26271023/67026207-e8a3f580-f139-11e9-91aa-0011945f0bda.png)
      Look forward to your reply ， thank you .

**Comment by h4yde** [comment]
@KangSpace Yes, it´s just the skeleton provided by nextcloud when you want to make an app.

**Comment by htgiang** [comment]
So great. Please share  treeview app for me. Many thanks
My contact: gianght249@gmail.com
